libraries/wallet/operation_printer.cpp

```diff
@@ -178,17 +185,14 @@ std::string operation_printer::operator()(const htlc_create_operation& op) const
    operation_result_printer rprinter(wallet);
    std::string database_id = result.visit(rprinter);
 
-   out << "Create HTLC to " << to.name
-         << " with id " << database_id
-         << " preimage hash: ["
-         << op.preimage_hash.visit( vtor )
-         << "] (Fee: " << fee_asset.amount_to_pretty_string( op.fee ) << ")";
```

Comment:
Fee removed?

Reply:
No, see line 195. Perhaps we should rename the fee method to something more like `print_fee`. It currently looks like a constructor IMO.